src/svc/nifid/schemas/Subscription.yml

```diff
@@ -10,25 +12,73 @@ properties:
   brokerHost:
     type: string
   brokerPort:
-    type: string
+    type: number
   brokerUsername:
     type: string
   topic:
     type: string
   dataFormat:
     type: string
   jsonMeasurementKey:
-    type: string
+    type: object
+    properties:
+      path:
+        type: string
+      name:
+        type: string
+      type:
+        type: string
   jsonFieldKeys:
-    type: string
+    type: array
+    items:
+      $ref: "./JsonSpec.yml"
   jsonTagKeys:
-    type: string
+    type: array
+    items:
+      $ref: "./JsonSpec.yml"
   jsonTimestamp:
-    type: string
-  stringLineProtocol:
-    type: string
+    type: object
+    properties:
+      path:
+        type: string
+      name:
+        type: string
+      type:
+        type: string
+  stringMeasurement:
+    type: object
+    properties:
+      pattern:
+        type: string
+  stringFields:
+    type: array
+    items:
+      type: object
+      properties:
+        pattern:
+          type: string
+        name:
+          type: string
```

Comment:
based on how we've defined our yml, jsonspec is it's own schema whereas the string/field & tag object is being defined inline, I suppose it's not an issue but just checking?

Reply:
fixed in commit 292ddc7